components/brave_wallet/browser/asset_discovery_manager.cc

```diff
@@ -421,4 +703,39 @@ absl::optional<SolanaAddress> AssetDiscoveryManager::DecodeMintAddress(
   return SolanaAddress::FromBytes(pub_key_bytes);
 }
 
+// static
+// Creates a URL like
+// https://api.simplehash.com/api/v0/nfts/owners?chains={chains}&wallet_addresses={wallet_addresses}
+GURL AssetDiscoveryManager::GetSimpleHashNftsByWalletUrl(
+    const std::string& account_address,
+    const std::vector<std::string>& chain_ids) {
+  if (chain_ids.empty() || account_address.empty()) {
+    return GURL();
+  }
+
+  std::string urlStr =
+      base::StringPrintf("%s/api/v0/nfts/owners", kSimpleHashUrl);
+  const base::flat_map<std::string, std::string>& simple_hash_chain_ids =
+      ToSimpleHashChainId();
```

Comment:
Can you confirm that this function will make it so we're only requesting for chainId's that are defined in this function and not for any arbitrary chainId?

Reply:
That's correct

Reply:
Thanks for confirming, sounds like a safe allow list approach. Resolving since there's no action on this.